src/libraries/System.Runtime/tests/TrimmingTests/System.Runtime.TrimmingTests.proj

```diff
@@ -18,6 +18,10 @@
       as a workaround while the linker bug is fixed. This bug has been logged in the
       linker repo here: https://github.com/mono/linker/issues/1285 -->
       <AdditionalArgs>/p:DebugType=Embedded</AdditionalArgs>
+      <!-- Justification: The implementation of StackFrame for Mono doesn't use
```

Comment:
Shouldn't the test still work though?

Reply:
This test wasn't passing on Wasm, not because it crashed but because the test checks that eventually the app got the line information out correctly, and in mono it looks like we are currently returning line number 0. Since I can't debug, it is not very straight forward by just looking at the code if this is just how StackFrame works on wasm or if it is due to some problem reading the pdb, but due to the nature of the test to be testing a specific annotation which doesn't exist on wasm, I opted for disabling it since the point of the test is not to ensure line info works on wasm, but instead just to test a specific annotation.

Reply:
cc: @steveisok @lewing might be able to help answer this question and also how to debug if needed in the future. What I know others did in the past and I've done as well to debug on wasm is literally print to the console 🗡️

Reply:
Let’s log a bug for the mono team to investigate and reference that bug here. In theory this should work, so we should find out why it doesn’t.

Reply:
Yeah I'd like to see this logged so we can refer back to it.  Broken debug info can definitely break our line numbers.

Reply:
https://github.com/dotnet/runtime/issues/48849 logged and added a comment on here as well in case we want to un-comment if/when that is fixed.